src/components/patch/PatchContainer.tsx

```diff
@@ -108,37 +128,56 @@ export class PatchContainer extends React.Component<Props, State> {
   }
 
   private filterItems(query: string) {
-    const filtered: UIVersion[] = [];
+    const filtered: UIPatch[] = [];
     if (query === "") {
       this.setState({
         isSearching: false
       });
       return this.state.allPatches;
     }
     Object.keys(this.state.allPatches).map(versionId => {
-      const patch = this.state.allPatches[versionId];
-      const description = patch.Version.message;
-      if (description.toLowerCase().indexOf(query.toLowerCase()) !== -1) {
+      const patch = this.state.allPatches[versionId] as UIPatch;
+      let queryField = "";
+      switch (this.state.searchType) {
+        case SearchType.description:
+          queryField = patch.Patch.Description
+          break;
+        case SearchType.project:
+          queryField = patch.Patch.Project
+          console.log(queryField);
```

Comment:
Take out these statements if you don't need them anymore

Reply:
Thanks, I totally missed these before I committed. I'll go ahead and remove any other stray logging statements I find.